homeassistant/components/blink/sensor.py

```diff
@@ -13,41 +15,48 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-SENSORS = {
-    TYPE_TEMPERATURE: ["Temperature", TEMP_FAHRENHEIT, DEVICE_CLASS_TEMPERATURE],
-    TYPE_WIFI_STRENGTH: [
-        "Wifi Signal",
-        SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
-        DEVICE_CLASS_SIGNAL_STRENGTH,
-    ],
-}
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key=TYPE_TEMPERATURE,
+        name="Temperature",
+        native_unit_of_measurement=TEMP_FAHRENHEIT,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+    ),
+    SensorEntityDescription(
+        key=TYPE_WIFI_STRENGTH,
+        name="Wifi Signal",
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        device_class=DEVICE_CLASS_SIGNAL_STRENGTH,
+    ),
```

Comment:
```suggestion
        entity_registry_enabled_default=False,
    ),
```

Reply:
Not sure about that one. If it should be a feature change, I would be happy open a new PR for it. This here should just cover the conversion.